cmd/gpud/main.go

```diff
@@ -5,9 +5,16 @@ import (
 	"os"
 
 	"github.com/leptonai/gpud/cmd/gpud/command"
+	"github.com/leptonai/gpud/version"
 )
 
 func main() {
+	// check for version flag before initializing the full app
+	if len(os.Args) == 2 && (os.Args[1] == "version" || os.Args[1] == "--version" || os.Args[1] == "-v") {
+		fmt.Println("gpud version", version.Version)
```

Comment:
we can try to remove this and test if --version is still ok?

Reply:
Done

```
time /tmp/gpud --version
gpud version 61be46d

real	0m0.030s
user	0m0.011s
sys	0m0.033s
```

Fast even without this change, removed.